controllers/windowsmachine_controller.go

```diff
@@ -76,6 +76,8 @@ type WindowsMachineReconciler struct {
 	// 		 in vSphere
 	//		 https://bugzilla.redhat.com/show_bug.cgi?id=1876987
 	platform oconfig.PlatformType
+	// username is the user that will be logged into on each VM in order to configure the Machine.
+	username string
```

Comment:
Why is `username` a property of the reconciler while IP address is not? `username` should be treated the same way. This should just be an input to `addWorkerNode`.

Reply:
I made it a property of the reconciler as username will be the same for all Machines reconciled, so it didnt make sense to me to figure out the username based on the platform, every time a Machine is added. IP address will be different for each Machine, so it is not comparable.

Reply:
Please think from a BYOH perspective where username can be different per instance. What will you do in that scenario?